src/components/emit-state-change.js

```diff
@@ -0,0 +1,33 @@
+AFRAME.registerComponent("emit-state-change", {
```

Comment:
Ah... perhaps that makes sense. This is a kind of "glue" component between super-hands' desire to add "hovered" states to the entity and our desire to play a sound with the `sound` component via an event listener.
I made each one separate because the pieces seemed simpler to me that way -- I expect to get rid of half of these because we only actually want to play sounds when we start a hover, not when we end a hover. 
If sticking them together avoids creating an extra component that isn't as free as we want it to be, perhaps the slight increase to complexity (if it's even correct to say taking them both together is more complex, which on second thought may be WRONG) is worth it.
Advise welcome.

Also I'd like to fix the code without a comment if it's not clear what it's doing, because I meant for it to be super straight forward. I get hover info via the `stateadded` listener and emit an event when the presence of that state changes.